haskell/src/Ligo/BaseDAO/TZIP16Metadata.hs

```diff
@@ -57,12 +70,27 @@ mkMetadataSettings msConfig = MetadataSettings
 -- without user's input.
 knownBaseDAOMetadata :: MetadataSettings -> Metadata (ToT Storage)
 knownBaseDAOMetadata settings = mconcat
-  [ version . fromString $ showVersion Paths.version
+  [ name "Base DAO"
+  , description "A generic smart contract on Tezos that enables a community to collectively govern resources, registries, or rules."
+  , version . fromString $ showVersion Paths.version
   , license $ License "MIT" Nothing
   , authors [Author "Serokell", Author "Tocqueville Group"]
   , homepage "https://github.com/tezos-commons/baseDAO"
-  , interfaces [tzip 17]
+  , interfaces
+      [ tzip 16
+      , tzip 21 -- 1728fcfe0ac90463ef15e6a994b6d6a15357e373
+      ]
   , views (baseDAOViews settings)
+  , keyValue @[MetadataAsset] "assets"
+      [ MetadataAsset
+          { maName = "Frozen Token"
+          , maDecimals = 0
+          , maDescription = "Frozen tokens are tokens that are required to be able to propose and vote on proposals."
+          , maShouldPreferSymbol = False
+          , maThumbnailUri = ""
+          , maCreators = ["TQ Tezos"]
+          }
+      ]
```

Comment:
@pasqu4le I've added more fields are you have suggested.

> for thumbnailUri I think we should ask on the shared slack channel, as some graphic already exist and this may very well have been uploaded to a fixed location already.

I couldn't find the thumbnail uri, so I leave it blank for now.

This is what the full metadata.json looks like, hope to get your review on this:

```
{
    "homepage": "https://github.com/tezos-commons/baseDAO",
    "interfaces": [
        "TZIP-16",
        "TZIP-21"
    ],
    "views": [
        {
            "implementations": [
                {
                    "michelsonStorageView": {
                        "returnType": {
                            "args": [
                                {
                                    "prim": "address",
                                    "annots": [
                                        "%address"
                                    ]
                                },
                                {
                                    "prim": "nat",
                                    "annots": [
                                        "%token_id"
                                    ]
                                }
                            ],
                            "prim": "pair"
                        },
                        "code": [
                            {
                                "prim": "CAR"
                            },
                            {
                                "prim": "CDR"
                            },
                            {
                                "prim": "CDR"
                            },
                            {
                                "prim": "CAR"
                            }
                        ]
                    }
                }
            ],
            "name": "governance_token",
            "pure": true,
            "description": "Return the address and token_id of the associated FA2 contract."
        },
        {
            "implementations": [
                {
                    "michelsonStorageView": {
                        "returnType": {
                            "prim": "nat"
                        },
                        "code": [
                            {
                                "prim": "CDR"
                            },
                            {
                                "prim": "CAR"
                            },
                            {
                                "prim": "CDR"
                            },
                            {
                                "prim": "CAR"
                            }
                        ]
                    }
                }
            ],
            "name": "GetCounter",
            "pure": true,
            "description": "Returns the next counter value with which a permit should be created."
        }
    ],
    "authors": [
        "Serokell",
        "Tocqueville Group"
    ],
    "name": "Base DAO",
    "version": "0.4.0.0",
    "license": {
        "name": "MIT"
    },
    "assets": [
        {
            "shouldPreferSymbol": false,
            "name": "Frozen Token",
            "creators": [
                "TQ Tezos"
            ],
            "thumbnailUri": "",
            "decimals": 0,
            "description": "Frozen tokens are tokens that are required to be able to propose and vote on proposals."
        }
    ],
    "description": "A generic smart contract on Tezos that enables a community to collectively govern resources, registries, or rules."
}
```

Reply:
Aside from the comments I left below it LGTM.

> I couldn't find the thumbnail uri, so I leave it blank for now.

It's ok, on second thought we probably don't want to use baseDAO's (or homebase's) logo for everyone :ok_hand: